addon/webextension/selector/ui.js

```diff
@@ -8,28 +8,6 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
 
   const { watchFunction } = catcher;
 
-  // The <body> tag itself can have margins and offsets, which need to be used when
-  // setting the position of the boxEl.
-  function getBodyRect() {
-    if (getBodyRect.cached) {
-      return getBodyRect.cached;
-    }
-    let rect = document.body.getBoundingClientRect();
-    let cached = {
-      top: rect.top + window.scrollY,
-      bottom: rect.bottom + window.scrollY,
-      left: rect.left + window.scrollX,
-      right: rect.right + window.scrollX
-    };
-    // FIXME: I can't decide when this is necessary
-    // *not* necessary on http://patriciogonzalezvivo.com/2015/thebookofshaders/
-    // (actually causes mis-selection there)
-    // *is* necessary on http://atirip.com/2015/03/17/sorry-sad-state-of-matrix-transforms-in-browsers/
-    cached = {top: 0, bottom: 0, left: 0, right: 0};
-    getBodyRect.cached = cached;
-    return cached;
-  }
```

Comment:
Since this patch contains more cleanup than new code, it warrants a quick explanation.  `getBodyRect()` always returns a bunch of zeroes (ignoring its own calculations), two of which are used in additions and subtractions later.  I just did the add/subtract zero math myself.